io.github.chidiwilliams.Buzz.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.chidiwilliams.Buzz
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: run-buzz.sh
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.secrets
+  - --device=dri
+  # TODO Switch 'all' to 'input' when it is widely available
+  #- --device=input
+  - --device=all
```

Comment:
why is `device=all` needed?

Reply:
Was intended for access to microphone for real time audio transcripts, but double checking the docs, that is provided by `--socket=pulseaudio`, will remove.